air_quality_monitoring/raqmn/raqmn_api/views.py

```diff
@@ -66,4 +66,35 @@ def get_data_by_uuid(request, uuid):
 		return_json = ast.literal_eval(storage.getvalue().replace('"',"'"))
 		return Response(return_json)
 
+@csrf_exempt
+@api_view(['GET'])
+def get_average_data_12_hrs(request, uuid):
+	"""
+	Returns the average of the data from the corresponding uuid of last 12 hrs
+	"""
+	if request.method == 'GET':
+		parser = HTMLParser()
+		uuid = parser.unescape(uuid)
+		query_url = 'http://localhost:8079/api/query'
+		query = "select data in (now -12h, now) where uuid='"+uuid+"'"
+		storage = StringIO()
+		curlObj = pycurl.Curl()
```

Comment:
Wait a minute - why are we using pycurl to post data instead of using a HTTP library like requests directly?
http://docs.python-requests.org/en/latest/